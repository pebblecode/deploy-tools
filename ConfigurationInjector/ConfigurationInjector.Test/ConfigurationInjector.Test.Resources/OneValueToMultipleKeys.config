<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <appSettings>
    <add key="db1.host" value="" />
    <add key="db1.name" value="" />
    <add key="db2.host" value="" />
    <add key="db2.name" value="" />
  </appSettings>

</configuration>
