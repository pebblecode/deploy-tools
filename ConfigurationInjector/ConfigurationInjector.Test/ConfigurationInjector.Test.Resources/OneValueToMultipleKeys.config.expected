<?xml version="1.0" encoding="utf-8"?>
<configuration>

  <appSettings>
    <add key="db1.host" value="localhost" />
    <add key="db1.name" value="sgp" />
    <add key="db2.host" value="localhost" />
    <add key="db2.name" value="sgp" />
  </appSettings>

</configuration>
